student/serializers.py

```diff
@@ -64,6 +68,7 @@ class Meta:
             'userFirstName',
             'userLastName',
             'userFullName',
+            'isAdvisor',
             'FacebookSignedUp',
```

Comment:
don't we want to courses they are registered for as well?

Reply:
and who their advisors are?

Reply:
I was thinking it would make more sense to have courses be gotten through a GET request since that's how it's also done for timetables. I think also adding advisor_names to the serializers was on the todo-list, haven't started that.

Reply:
ah makes sense for courses.